swift/ITSClient/Sources/ITSMobility/Models/ETSI.swift

```diff
@@ -0,0 +1,105 @@
+/*
+ * Software Name : ITSClient
+ * SPDX-FileCopyrightText: Copyright (c) Orange SA
+ * SPDX-License-Identifier: MIT
+ *
+ * This software is distributed under the MIT license,
+ * see the "LICENSE.txt" file for more details or https://opensource.org/license/MIT/
+ *
+ * Software description: Swift ITS client.
+ */
+
+import Foundation
+
+/// A structure to make conversion for ESTI units.
+struct ETSI {
+    private static let deciMicroDegreesFactor: Double = 10_000_000
+    private static let etsiUnixEpochDifference = 1072915200
```

Comment:
Best would be if that could be made an actual computation, something like:
```
etsiUnixEpochDifference = UnixTime("2004-01-01T00;00:00+00:00").as_seconds()
```
(you get the idea).

At the very least, add a comment to explain how it was computed.

Reply:
Swift date represents a time interval relative to an absolute reference time in seconds without taking into account timezone or calendar system. To do the same thing as your proposal, we must use a DateFormatter to express a date with a string. Unfortunately, it's a bit expensive and we need more code. I suggest adding comments for these reasons if you agree.

Reply:
ACK for the comment.